src/GitTfs/Commands/Clone.cs

```diff
@@ -171,6 +171,16 @@ private void VerifyTfsPathToClone(string tfsRepositoryPath)
 
                 if (!remote.Tfs.CanGetBranchInformation)
                     return;
+
+                //TIM C: added this so it would still continue on for even folders. As we have a desire to clone those as well, but only if the strategy is None.
+                if (_fetch.BranchStrategy == BranchStrategy.None)
+                {
+                    if (remote.Tfs.ValidateTfsFolder(tfsRepositoryPath))
+                        return;
+                    else
+                        throw new GitTfsException($"The TFS path ({tfsRepositoryPath}) does not appear to be a valid folder.");
+                }
+
```

Comment:
Can't we just do this instead of this whole block including the previous 
```
if (!remote.Tfs.CanGetBranchInformation)
                    return;
```
```suggestion
                if (_fetch.BranchStrategy == BranchStrategy.None || !remote.Tfs.CanGetBranchInformation)
                    return;
```

E.g. is there a reason why we need to care if we have a folder or a file specified by the user?
I would assume that we need to handle the _"user gave us a file"_ case anyhow if `CanGetBranchInformation`  returns `false`. This would also completely get rid of that newly introduced method `ValidateTfsFolder`.
But I have to admit I haven't checked... so take this with a grain of salt

Reply:
I looked at the code for that function and I am not sure it fulfills the needs: 
`        public bool CanGetBranchInformation
        {
            get { return !Is2008OrOlder; }
        }`

I am not against getting rid of the ValidateTfsFolder and using IsExistingInTfs.  That leaves the choice up to the user if they decide to clone a file or a deleted folder. It is not something I would want to do, but that would give them the choice.

Reply:
I meant combine the check first for `_fetch.BranchStrategy == BranchStrategy.None` and if that is true don't even bother to check the `CanGetBranchInformation`. Simply return in both cases.

Reply:
Oh, I see. That is quite acceptable to me.